zeroconf/test.py

```diff
@@ -860,6 +861,20 @@ def test_cache_empty_does_not_leak_memory_by_leaving_empty_list(self):
         cache.remove(record2)
         assert 'a' not in cache.cache
 
+    def test_cache_empty_multiple_calls_does_not_throw(self):
```

Comment:
Does this patch actually fix something that's verified by this test? Let's keep this out of this pull request if that's not the case.

Reply:
I'll pull it out.  It should probably go in a new PR after this merges